api/workAllocation2/index.ts

```diff
@@ -389,6 +391,7 @@ export async function postTaskSearchForCompletable(req: EnhancedRequest, res: Re
       'case_type': req.body.searchRequest.caseTypeId,
       'event_id': req.body.searchRequest.eventId,
     };
+    console.log('ravi reqBody', reqBody)
```

Comment:
Can we remove console.log statement please?

Reply:
yes

Reply:
Remove